ai21/models/chat/chat_message.py

```diff
@@ -1,8 +1,30 @@
 from __future__ import annotations
 
+from typing import Literal, List
+
 from ai21.models.ai21_base_model import AI21BaseModel
+from ai21.models.chat.tool_call import ToolCall
 
 
 class ChatMessage(AI21BaseModel):
```

Comment:
As mentioned above, maybe worth using `TypedDict` instead for request objects, allowing us to omit all `| None = None`. If we do decide to go in this direction, then the `message` property for `ChatCompletionResponseChoice` would have to be a `AI21BaseModel` called something like `ChatCompletionAssistantMessage`